test/test_kubeclient.rb

```diff
@@ -200,7 +198,6 @@ def test_nonjson_exception
       client.get_services
     end
 
-    assert(exception.message.include?('Not Found'))
```

Comment:
With `rest_client` the exceptions's `message` would include the reason phrase. This is `rest_client`-specific, `faraday` or `net/http` does not provide this for us.